db/seeds.rb

```diff
@@ -1,9 +1,9 @@
 require 'csv'
 require './app/models/station'
 
-CSV.foreach('db/csv/station.csv', {headers: true, header_converters: :symbol}) do |row|
-  Station.create(name:              row[:name],
-                 city:              row[:city],
-                 dock_count:        row[:dock_count],
-                 installation_date: row[:installation_date])
-end
+  CSV.foreach('db/csv/station.csv', {headers: true, header_converters: :symbol, converters: :numeric}) do |row|
```

Comment:
What does converters: :numeric do?

Reply:
It converts strings that look like numbers in the csv file to actual numbers

Reply:
like the lat/long and the dock_count

Reply:
That's pretty cool, nice work getting the date setup